includes/template-functions.php

```diff
@@ -399,10 +399,7 @@ function give_add_post_class( $classes, $class = '', $post_id = '' ) {
  * @return string
  */
 function give_get_placeholder_img_src() {
-
-	$placeholder_url = '//placehold.it/600x600&text=' . urlencode( esc_attr__( 'GiveWP Placeholder Image', 'give' ) );
-
-	return apply_filters( 'give_placeholder_img_src', $placeholder_url );
+	return apply_filters( 'give_placeholder_img_src', GIVE_PLUGIN_URL . 'assets/dist/images/give-placeholder.png' );
```

Comment:
@bradp Any reason not using `esc_url` here?

Reply:
URL is built from pieces we control in the plugin, so we shouldn't need to escape it.